nearlib/signing/simple_key_store_signer.js

```diff
@@ -17,13 +17,16 @@ class SimpleKeyStoreSigner {
      * @param {string} senderAccountId
      */
     async signTransactionBody(body, senderAccountId) {
+        return this.signHash(new Uint8Array(sha256.array(body)), senderAccountId);
+    }
+
+    async signHash(hash, senderAccountId) {
```

Comment:
do we have a test for this? let's add one

Reply:
Tracking here https://github.com/nearprotocol/nearcore/issues/573